changelog/issue-5773.md

```diff
@@ -0,0 +1,6 @@
+audience: users
+level: minor
```

Comment:
Think this is minor? Seems a little more like a patch bump to me 🤷🏻‍♂️

Reply:
yes! will update